client/src/components/landing_page/ExploreMapSection.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <LandingPageSection class='section'
+                      :messages='messages.MapInfo'
+                      :styles='styles.DEFAULT_STYLE'>
+    <img class='static-map'
+         alt=''
+         :src='require(`@/assets/media/static-map.png`)'>
+    <button class='explore-map-button' @click='goToMap'>
+      {{ messages.MapInfo.CTA_BUTTON }}
+    </button>
+  </LandingPageSection>
+</template>
+
+<script lang='ts'>
+import { defineComponent } from 'vue';
+import LandingPageSection from './LandingPageSection.vue';
+import * as messages from '@/assets/messages/landing';
+import { LandingPageStyles as styles } from '@/assets/styles/style_props';
+import { router } from '@/router';
+
+/**
+ * Content for 'Explore map' section of the landing page.
+ */
+export default defineComponent({
+  name: 'ExploreMapSection',
+  components: {
+    LandingPageSection,
+  },
+  data() {
+    return {
+      messages,
+      styles,
+    };
+  },
+  methods: {
+    goToMap(): void {
```

Comment:
I can edit my PR as well but just so we are consistent throughout. Do you want to call routing changes "goTo..." or "navigateTo"

Reply:
I'll switch!